[2025-01-08T07:33:38.855+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-08T07:33:38.863+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9863800249_2025-01-08T07-32-29-461421Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-08T07:32:29.461421+00:00 [queued]>
[2025-01-08T07:33:38.866+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9863800249_2025-01-08T07-32-29-461421Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-08T07:32:29.461421+00:00 [queued]>
[2025-01-08T07:33:38.867+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-08T07:33:38.872+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): check_if_callerid_is_present_in_calllogs_table> on 2025-01-08 07:32:29.461421+00:00
[2025-01-08T07:33:38.876+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=7544) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-08T07:33:38.877+0000] {standard_task_runner.py:72} INFO - Started process 7545 to run task
[2025-01-08T07:33:38.877+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9863800249_2025-01-08T07-32-29-461421Z', 'check_if_callerid_is_present_in_calllogs_table', 'scheduled__2025-01-08T07:32:29.461421+00:00', '--job-id', '986', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpdeq5opek']
[2025-01-08T07:33:38.878+0000] {standard_task_runner.py:105} INFO - Job 986: Subtask check_if_callerid_is_present_in_calllogs_table
[2025-01-08T07:33:38.899+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9863800249_2025-01-08T07-32-29-461421Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-08T07:32:29.461421+00:00 [running]> on host 680cb867086b
[2025-01-08T07:33:38.936+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9863800249_2025-01-08T07-32-29-461421Z' AIRFLOW_CTX_TASK_ID='check_if_callerid_is_present_in_calllogs_table' AIRFLOW_CTX_EXECUTION_DATE='2025-01-08T07:32:29.461421+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-08T07:32:29.461421+00:00'
[2025-01-08T07:33:38.936+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-08T07:33:38.940+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2025-01-08T07:33:38.944+0000] {sql.py:544} INFO - Running statement: SELECT * FROM call_logs WHERE caller_id = 'demo-scheduled-from-ui-global8c6bf124-9aae-4f36-9d6f-519c26887c98' limit 1;, parameters: None
[2025-01-08T07:33:38.947+0000] {sql.py:553} INFO - Rows affected: 0
[2025-01-08T07:33:38.947+0000] {python.py:240} INFO - Done. Returned value was: unanswered
[2025-01-08T07:33:38.957+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-08T07:33:38.957+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9863800249_2025-01-08T07-32-29-461421Z, task_id=check_if_callerid_is_present_in_calllogs_table, run_id=scheduled__2025-01-08T07:32:29.461421+00:00, execution_date=20250108T073229, start_date=20250108T073338, end_date=20250108T073338
[2025-01-08T07:33:38.970+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-08T07:33:38.982+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-08T07:33:38.982+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
